lib/puppet/face/facts.rb

```diff
@@ -1,5 +1,24 @@
 require 'puppet/indirector/face'
 require 'puppet/node/facts'
+require 'puppet/util/fact_dif'
+
+EXCLUDE_LIST = %w[ fips_enabled facterversion identity\.gid identity\.privileged identity\.uid
+load_averages\.15m load_averages\.1m load_averages\.5m memory\.swap\.available_bytes
+memory\.swap\.capacity memory\.swap\.total_bytes memory\.swap\.used_bytes memory\.swap\.available
+memory\.system\.available memory\.system\.available_bytes memory\.system\.capacity memory\.swap\.used
+memory\.system\.total_bytes memory\.system\.used memory\.system\.used_bytes memoryfree memoryfree_mb
+memorysize_mb mountpoints\..* mtu_.* networking\.interfaces\..*\.mtu networking\.mtu
+os\.selinux\.enabled partitions\..*\.filesystem partitions\..*\.size_bytes partitions\..*\.mount
+partitions\..*\.uuid physicalprocessorcount processorcount processors\.count
+processors\.physicalcount selinux swapfree_mb swapsize_mb system_uptime\.days system_uptime\.hours
+system_uptime\.seconds uptime_days uptime_hours uptime_seconds clientnoop swapfree
+disks\..*\.size_bytes hypervisors\.lpar\.partition_number mountpoints\..*\.capacity
+processors\.speed serialnumber hypervisors\.xen\.privileged os\.release.\minor
+operatingsystemrelease os\.release\.full os\.distro\.description filesystems
+sp_uptime system_profiler\.uptime os\.release\.minor
+hypervisors\.zone\..* system_uptime\.uptime uptime hypervisors\.ldom\..* ldom_.*
+boardassettag dmi\.board\.asset_tag is_virtual kernelmajversion lsbmajdistrelease zones virtual
+blockdevice_.*_vendor blockdevice_.*_size]
```

Comment:
I don't think we should be skipping facts that are not volatile, eq: kernelmajversion. Both cfacter and facter 4 should return the same value for that facts.